bin/millau/runtime/src/lib.rs

```diff
@@ -217,12 +198,18 @@ impl frame_system::Trait for Runtime {
 	// TODO: update me (https://github.com/paritytech/parity-bridges-common/issues/78)
 	/// Weight information for the extrinsics of this pallet.
 	type SystemWeightInfo = ();
+	/// boop
```

Comment:
boop? :)

Reply:
Ah this is embarrassing 😅 I meant to write actual docs